### ISSUE TITLE
Have a link up top that provides a link to the original post/publication (e.g., medium, law review article, etc)

### ISSUE THREAD

**Issue by johnquiwa** [yungtiec/bkp-alpha#184]

**Comment by johnquiwa** [comment]
@flyingdutchnick Could we just add these to the beginning of the markdown for each document? i.e. _The original post can be found at [link]_?

**Comment by flyingdutchnick** [comment]
Yes, to close out this issue, I recommmend we don’t write any code but just come up with a standard template or way of presenting it that is reusable and makes it look consistent across documents.
